test/lib/Launcher.ts

```diff
@@ -37,7 +37,7 @@ export class Launcher {
 
         for (let i = 0; i < count; i++) {
             const port = await getPort();
-            const accountName = await this.randomString();
+            const accountName = `${i + 1}-${await this.randomString()}`;
```

Comment:
?

Reply:
Oh, somehow my code comment got lost it is there to improve debugging the tests when accessing the DB to see what client is related to which DB

Reply:
Oh, I see. When debugging one test suite at once this gets really handy, right? I don't think we need a comment for that, I'm ok with the change.